frontend/website/pages/hardware-wallet.mdx

```diff
@@ -19,9 +19,81 @@ The following debian packages are needed to use hardware wallets on linux:
 sudo apt install libusb-1.0-0-dev libudev1 libudev-dev
 ```
 
+### Make sure your computer is recognising the Ledger device
+
+- Plug in the Ledger device, unlock it with your passcode, and go to the menu. Don't open any app,
+- Run `lsusb`, and see if the device is listed -- it will have ID `2c97:0001`
+- If the Ledger device is not listed, follow the suggestions [here](https://support.ledger.com/hc/en-us/articles/115005165269-Fix-connection-issues).
+
+### Build/download the Coda Ledger application
+
+Either download the precompiled app binary from [here](...).
+
+#### System-wide installation
```

Comment:
Can we move the Build-from-source instructions to the bottom of the docs -- that way people that want to download a precompiled binary won't have to scroll past the build instructions before getting to the `Using the wallet` section.